src/LuauRenderer/nodes/types/renderTypeFunction.ts

```diff
@@ -0,0 +1,22 @@
+import luau from "LuauAST";
+import { render, RenderState } from "LuauRenderer";
+
+export function renderTypeFunction(state: RenderState, node: luau.TypeFunction) {
+	let params: string;
+
+	if (node.parameters !== undefined) {
+		const rendered = luau.list.mapToArray(node.parameters, expr => render(state, expr));
+		if (node.dotDotDot) {
+			rendered.push(`...${render(state, node.dotDotDot)}`);
+		}
+		params = rendered.join(", ");
+	} else {
+		if (node.dotDotDot) {
```

Comment:
This can be simplified to an `else if`.
I wonder if it makes more sense to always use `rendered` and do `const params = rendered.join(", ") || "...any"`.
Or maybe make the parameters required? I'm not sure a fallback to `...any` makes sense for Luau-AST.